asreview/webapp/src/SettingsDialog.js

```diff
@@ -102,6 +102,17 @@ export default function SettingsDialog(props) {
                 />
               </ListItemSecondaryAction>
             </ListItem>
+            <ListItem>
+              <ListItemText id="switch-list-label-key" primary="Keyboard shortcut" />
```

Comment:
"Enable keyboard shortcuts"?

Reply:
I'd say "Enable" is not necessary, as it is clear to see from the switch icon when it is toggled on or off. If it is "Enable xxx", it should be changed to "Disable xxx" when it is toggled on. So, the same logic applies to that "Undo Enable" could be "Undo labeling".
Maybe we also need a help text of settings dialog to explain these features briefly.

Reply:
Agree.